pirogue/utils.py

```diff
@@ -440,7 +443,8 @@ def value(col):
 
     return """UPDATE {s}.{t}{a} SET
 {indent}    {cols}
-{indent}  WHERE {where_clause};""".format(
+{indent}  WHERE {where_clause}
+{indent}  ){returning};""".format(
```

Comment:
there is an extra `)` here or I am missing something?

Reply:
you are correct